_docs/demo/setup.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: page
+title: Setup
+permalink: /docs/demo/setup/
+category: Demo
+order: 1
+---
+
+While Amalgam8 supports multi-tenancy, for the sake of simplicity, this
+demo will setup Amalgam8 in single-tenant mode.
+
+## Download
+
+Download and unzip the latest release of Amalgam8 demos from the
+[Amalgam8 Releases](https://www.github.com/amalgam8/amalgam8/releases)
+page.
+
+## Requirements
+
+* [Docker](https://www.docker.com/products/docker#/)
+  You need `docker` 1.10 or higher and `docker-compose` 1.5.1 or higher.
+
+* [Python 2.7.9](https://www.python.org/downloads/) or higher. If you are
+  using Linux or Mac OS X, you should already have the requisite version of
+  Python in your system.
+
+### Additional Requirements
+
+* If you are planning to use Kubernetes, then you need a
+  working kubernetes installation. To install kubernetes locally, checkout
+  [minikube](https://github.com/kubernetes/minikube).
+  
+  * If you are using Google Cloud Platform, setup the
+    [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+* If you are planning to run the demos on IBM Bluemix, you need to have
+  the [CF CLI 6.12.0 or later](https://github.com/cloudfoundry/cli/releases), and
+  the [Bluemix CLI 0.3.3 or later](https://clis.ng.bluemix.net/).
+
+With the requirements out of the way, lets move on to the next stage:
+running Amalgam8.
+
+## Setting up the Control Plane
+
+The Amalgam8 Control Plane consists of two components: the
+[service registry](/docs/registry/) and the
+[route controller](/docs/controller/).  In addition to these two
+components, for the purposes of this demo, a stock ELK stack is also
+included as part of the control plane deployment scripts, to collect logs
+from the microservices in the application.
+
+The commands to deploy the control plane for different environments are as
+follows:
+
+_Docker Compose_
+  
+```bash
+docker-compose -f examples/docker-controlplane.yaml up -d
+```
+
+_Kubernetes_ on localhost or on Google Cloud
+
+```bash
+kubectl create -f examples/k8s-controlplane.yaml
+```
+
+_IBM Bluemix_
+
+1. Login to Bluemix and initialize the container environment using 
+
+   ```
+   bluemix login
+   bluemix ic init
+   ```
+
+1. Create Bluemix routes (DNS names) for the registry, controller and the bookinfo app's gateway:  
+
+   ```bash
+   cf create-route <your bluemix space> mybluemix.net -n <your registry route>
+   cf create-route <your bluemix space> mybluemix.net -n <your controller route>
+   ```
+
+   where `<your bluemix space>` is the name of your Bluemix space and
+   `<your route ...>` is a unique route name for `registry` and
+   `controller`. For example `my-space-name-a8-registry` and 
+   `my-space-name-a8-controller`. Make a note of the route names you
+   choose for the next step.
+
+
+1. Download the scripts to automate Bluemix deployment from this
+   [here](https://url_to_bluemix_scripts).
```

Comment:
User has already downloaded the demos release, no?  Just tell them to open the bluemix folder?